usage-information.md

```diff
@@ -9,7 +9,7 @@ Please see [development-methodology.md](development-methodology.md) for the appr
 
 Please see [installation.md](installation.md) for installation and usage instructions.
 
-An API for the converter is being developed which will allow conversion of notices via HTTPS requests.
+An API for the converter will be made available soon which will allow conversion of notices via HTTPS requests.
```

Comment:
An API for the converter is now available ?

Reply:
The API is not publicised yet. I need to coordinate with Victor and Karl about this.